chainstate/src/detail/transaction_verifier/mod.rs

```diff
@@ -598,6 +678,9 @@ impl<'a, S: BlockchainStorageRead> TransactionVerifier<'a, S> {
                         .unspend(outpoint.output_index())
                         .map_err(ConnectTransactionError::from)?;
                 }
+
+                // Remove issued tokens
+                self.remove_tokens(tx)?;
```

Comment:
Is this supposed to be `unregister_token_issuance()`?

Reply:
Cool, I've just renamed it.